persistence/persistence-api/src/main/resources/applicationContext-ehcache.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:context="http://www.springframework.org/schema/context"
+    xmlns:cache="http://www.springframework.org/schema/cache"
+    xsi:schemaLocation="
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd
+    http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
+    http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-4.0.xsd">
+
+    <cache:annotation-driven cache-manager="cacheManager" key-generator="customKeyGenerator"/>
+
+    <bean id="cacheEnabledConfig" class="org.cbioportal.persistence.CacheEnabledConfig">
+        <constructor-arg value="${ehcache.cache_enabled}"/>
```

Comment:
also here good to set the default (apart from in `portal.properties.EXAMPLE`)

Reply:
done